docs/src/routes/docs/components/autocomplete-field.svx

```diff
@@ -0,0 +1,133 @@
+    link: ./docs/components/autocomplete-option
+---
+
+<script>
+  import { AutocompleteField } from 'attractions';
+  import { ChevronDownIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+
+  let selection = [];
+</script>
+
+
+# AutocompleteField
+
+A plain unstyled input element with a dropdown for suggestions.
+
+The [==`Autocomplete`==](./docs/components/autocomplete) component uses it to provide a full-featured autocomplete solution with chips for selection, it may be better if you don't intend to display the selection on your own.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase">
+    <AutocompleteField {getOptions} bind:selection />
+    <pre>{JSON.stringify(selection, null, 2)}</pre>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+
+  let selection = [];
+</script>
+
+<AutocompleteField {getOptions} bind:selection />
+<pre>{JSON.stringify(selection, null, 2)}</pre>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| ==**`getOptions`**==&nbsp;<span class="required">*</span> | `async function*` | An async generator of suggestions. <br /> Receives the input from the text field and is expected to yield promises that resolve to arrays of objects `{ name, details }`. |
+| **`selection`** | `Array` | The current selection as an array of objects `{ name, details }`. <br /> Can be used to set the selection programmatically. |
+| **`minSearchLength`** | `number` | The minimum amount of characters to call ==`getOptions`==. <br /> 3 by default. |
+| **`maxOptions`** | `number` | The maximum amount of options than can be selected. Set to `null` (the default) for no restrictions. |
+| **`searchQuery`** | `number` | The current value of the text field. Can be used to control the query programmatically. |
+| **`optionComponent`** | Svelte component | The component used to render a suggestion in the list. <br /> Receives two props: the `option` object (`{ name, details }`) and the current `query`. <br /> Expected to dispatch `click` events on selection. <br /> Defaults to [==`AutocompleteOption`==](./docs/components/autocomplete-option). |
+| **`focus`** | `boolean` | Allows to programmatically control the suggestions dropdown. |
```

Comment:
How about renaming this to `active`, or at least explain what does it control about the dropdown? The description is not really clear and I had to go check the source code